_posts/2022-01-17-changelog-163.markdown

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #163"
+date: 2022-01-17 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8018] Add a widget for the [external linter](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-analysis.html#call-external-linter).
+Widget indicates whether the linter is turned on. Also, the widget shows a notification when linter significantly affects the code analysis performance.
+The external linter settings were moved to `Preferences | Languages & Frameworks | Rust | External Linters`.
+If necessary, the maximum duration for the external linter can be changed via `Registry... | org.rust.external.linter.max.duration` option
```

Comment:
Let's not mention `Registry` here if possible since it's more about workaround for unexpected cases than real setting. If we decide that users need this setting, we will move it into user-visible panel